.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: required
 dist: trusty
 language: node_js
 node_js:
-- '6.9.1'
```

Comment:
The `engines` could probably be `>=6.0`, we just need `let` and `const` support.